tests/conftest.py

```diff
@@ -4,8 +4,36 @@
 import sys
 from os.path import abspath, dirname, join
 
+import pytest
+
 
 # allow having multiple repository checkouts and not needing to remember to rerun
 # 'pip install -e .[dev]' when switching between checkouts and running tests.
 git_repo_path = abspath(join(dirname(dirname(__file__)), "src"))
 sys.path.insert(1, git_repo_path)
+
+# import local modules after fixing up sys.path
+if 1:  # flake be quiet
+    from transformers.utils.logging import (
+        logging_level_str_to_code,
+        logging_levels_as_strings,
+        set_global_logging_level,
+    )
+
+
+def pytest_addoption(parser):
+    parser.addoption(
+        "--loglevel",
+        type=str,
+        default=False,
+        choices=logging_levels_as_strings(),
```

Comment:
I think `log_levels.keys()` is better if it's only used here.

Reply:
are you suggesting moving the whole dict here? then yes, `.keys()` would work. But then we end up with a lot of code in the end-user code, which belongs to the logging package. 

And I can see how this function can be useful to others if they want to do the same in their CLI scripts.

Reply:
> are you suggesting moving the whole dict here? then yes, `.keys()` would work. But then we end up with a lot of code in the end-user code, which belongs to the logging package. 
> 
> 
> 
> And I can see how this function can be useful to others if they want to do the same in their CLI scripts.

Ok, that makes sense